### DISCUSSION TITLE
SU2 was not compiled for an AVX-capable architecture

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#1940] [Q&A]
**Desktop (please complete the following information):**
 - OS: WSL (Ubuntu22.04)
 - C++ compiler and version: g++ 11.3.0
 - MPI implementation and version: 4.0
 - SU2 Version: v7.5.1

Hello：
My computer have 2 epyc cpus，when I run QuickyStart ，
it warns：SU2 was not compiled for an AVX-capable architecture. Performance could be better。
My compile configuration：
./meson.py build -Denable-autodiff=true  -Denable-directdiff=true -Denable-pywrapper=true 

How can I solve it？
Thanks！

**Comment by pcarruscag** [comment]
https://www.cfd-online.com/Forums/su2-installation/244862-su2-not-installed-avx-architecture.html

**Reply by Zcaic** [reply]
Thanks a lot! I solve it by typing
export CXXFLAGS="-march=native -funroll-loops -O2"
before ./meson.py build
